src/psyclone/domain/common/transformations/kernel_module_inline_trans.py

```diff
@@ -174,9 +175,16 @@ def validate(self, node, options=None):
 
     @staticmethod
     def _prepare_code_to_inline(code_to_inline):
-        ''' Prepare the PSyIR tree to inline by bringing in to the subroutine
+        '''Prepare the PSyIR tree to inline by bringing in to the subroutine
         all referenced symbols so that the implementation is self contained.
 
+        issue #2271 will improve this method and could potentially
```

Comment:
Can you do s/issue/TODO/ in this line to make it easier to track.

Reply:
Oops